lib/loadtest.js

```diff
@@ -139,14 +139,29 @@ class Operation {
 	 */
 	startClients() {
 		const url = this.options.url;
+		const strBody = JSON.stringify(this.options.body);
+		function replace(str, oldToken, newToken) {
```

Comment:
I don't see the need for this function, why not just use `str.replace()` directly?